pyphare/pyphare/pharein/initialize/hybrid.py

```
@@ -60,8 +65,22 @@ def populateDict(sim):
+        #     pass
+
         for item in sim.electrons.dict_path():
             if isinstance(item[1], str):
                 add_string("simulation/" + item[0], item[1])
-            else:
+            elif isinstance(item[1], float):
                 add_double("simulation/" + item[0], item[1])
+            elif isinstance(item[1], Callable):
+                addInitFunction("simulation/" + item[0], fn_wrapper(item[1]))
+            else:
+                raise ValueError(f"acceptable entries should be int, float or collable")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)